themes/hugo-future-imperfect/layouts/partials/footer.html

```diff
@@ -38,8 +38,23 @@
         {{ end }}
       {{ end }}
     {{ end }}
-
+    
+    <script type="text/javascript" src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>
```

Comment:
`<script type="text/javascript"
        src="http://cdn.mathjax.org/mathjax/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML">
</script>`

Reply:
Should not we add the `mathjax-style.css` also ?

Reply:
I only added what I need to fix the missing math.

Reply:
allright.